src/braket/aws/aws_device.py

```diff
@@ -582,6 +587,34 @@ def ports(self) -> dict[str, Port]:
         self._update_pulse_properties()
         return self._ports or {}
 
+    def emulator(self, local: bool = True) -> Emulator:
+        """
+        A device emulator mimics the restrictions and noise of the AWS QPU by validating and
+        compiling programs before running them on a simulated backend. An emulator can be used
+        as a soft check that a program can run the target AwsDevice.
+
+        Returns:
+            Emulator: An emulator for this device, if this is not a simulator device. Raises an
+            exception if an emulator is requested for a simulator device.
+        """
+        if self._arn in [simulator_enum.value for simulator_enum in Devices.Amazon]:
+            raise ValueError(
+                "Creating an emulator from a Braket managed simulator is not supported."
+            )
+        if not local:
+            raise ValueError("local can only be True.")
+
+        self._emulator = LocalEmulator.from_device_properties(self.properties)
+
+        # Add device-specific gate value validators
+        if "Ankaa" in self.name:
+            self._emulator.add_pass(AnkaaRxValidator())
+
+        if "Aria" in self.name:
+            self._emulator.add_pass(AriaMSValidator())
```

Comment:
More robust to instead have a top-level map
```python
_DEVICE_PASEES = {
    Devices.IonQ.Aria1: [AriaMSValidator()],
    Devices.IonQ.Aria2: [AriaMSValidator()],
    Devices.Rigetti.Ankaa3: [AnkaaRxValidator()]
}
```
You never know, a QHP _could_ introduce different behavior for a new QPU of the same class; as well, it's safer to use defined types rather than strings.